src/main/java/duke/UI/UI.java

```diff
@@ -0,0 +1,67 @@
+package duke.UI;
```

Comment:
Should package name contain any capitalized alphabets?
```suggestion
package duke.ui;
```